stress/buildscripts/push

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+IMAGEID=$( docker images -q litmuschaos/cpu:latest)
+
+if [ ! -z "${DNAME}" ] && [ ! -z "${DPASS}" ]; 
+then 
+  docker login -u "${DNAME}" -p "${DPASS}"; 
+  #Push to docker hub repository with latest tag
+  docker tag ${IMAGEID} litmuschaos/cpu:latest
```

Comment:
why do we tag if it is already latest?

Reply:
Yes, but they also did it for forkbomb anyways i'll remove it.